src/app/pages/splash/containers/splash-page.component.ts

```diff
@@ -73,9 +73,13 @@ export class SplashPageComponent {
             const timeElapsed = Date.now() - lastUpdate
             if (timeElapsed > DefaultNotificationRefreshTime || !lastUpdate) {
               console.log('[SPLASH] Scheduling Notifications.')
-              this.notificationService.setNextXNotifications(
-                DefaultNumberOfNotificationsToSchedule
-              )
+              return this.notificationService
+                .cancelNotifications()
```

Comment:
Again this is not required here until the protocol, Time zone or Lang is changed but those cases are handled elsewhere. These have quite some overhead on the network and especially on the server.

Reply:
I've made the changes.